olive/systems/docker/utils.py

```diff
@@ -19,19 +19,11 @@
 logger = logging.getLogger(__name__)
 
 
-def create_config_file(
-    tempdir, model_config: "ModelConfig", metrics: List["Metric"], container_root_path: Path, model_mounts: dict
-):
-    model_json = model_config.to_json(check_object=True)
-    for k, v in model_mounts.items():
-        model_json["config"][k] = v
-
+def create_config_file(tempdir, config: dict, container_root_path: Path, model_mounts: dict):
```

Comment:
Can we also rename the `tempdir` lik what you did in `docker.system.py`?

Reply:
will change to workdir